### DISCUSSION TITLE
OrientDB over SSL

### DISCUSSION THREAD

**Discussion by ComeOutToPlay** [orientechnologies/orientdb#9728] [General]
Does anybody made it work OrientDb over SSL following the keytool instructions at [Using SSL with OrientDB](http://orientdb.com/docs/3.1.x/security/Using-SSL-with-OrientDB.html)?

It works for me but I get a "Not Secure" warning at chrome when navigating to https://localhost:2434

Any orientation will be appreciated.

Thanks.

**Comment by frbayart** [comment]
@ComeOutToPlay , your certificate is it defined for `localhost` domain?
